railties/lib/rails/generators/rails/app/app_generator.rb

```diff
@@ -127,7 +127,7 @@ def config
         template "routes.rb" unless options[:update]
         template "application.rb"
         template "environment.rb"
-        template "bundler-audit.yml"
+        template "bundler-audit.yml" unless options[:skip_bundler_audit]
```

Comment:
```suggestion
        template "bundler-audit.yml" unless skip_bundler_audit?
```

Is there a reason to use `options` here?